### DISCUSSION TITLE
How does the notification system work?

### DISCUSSION THREAD

**Discussion by Timoyoungster** [d99kris/nchat#256] [Q&A]
Hello,
how does nchat handle sending notifications to the user?
Is it just adding the star symbol for each chat, or can it also send notifications via a notification manager like meko?
I am using hyprland with meko and apps like discord are able to show messages via meko but nchat doesn't.
I'm wondering whether this is a configuration issue on my end or nchat just doesn't support that.

Thanks in advance!

**Comment by Bellavene** [comment]
As usual
`desktop_notify_command=`

**Comment by d99kris** [comment]
As @Bellavene pointed out, you can configure [desktop_notify_command](https://github.com/d99kris/nchat#desktop_notify_command) - example is provided there. You also need to enable `desktop_notify_active` and/or `desktop_notify_inactive`. If for some reason you are not getting notifications, I'd suggest to try to run your command set for `desktop_notify_command` manually in the terminal and try troubleshoot from there.

I'm not familiar with `meko` though, so I'm not sure if it supports notifications via `notify-send` or whether it has some other command line tool for posting notifications.

**Reply by Timoyoungster** [reply]
Thank you, that's the setting I was missing!
